core/src/main/scala/filodb.core/memstore/TimeSeriesPartition.scala

```diff
@@ -242,7 +242,7 @@ extends ChunkMap(memFactory, initMapSize) with ReadablePartition {
         // FIXME Two traces below to debug SEGV seen on read of info.id during downsampling
         // Remove after debug is done
         _log.trace(s"Preparing to flush part $stringPartition")
```

Comment:
This log line is redundant.  stringPartition is already printed below.

Reply:
The reason for two lines is commented above.

```// FIXME Two traces below to debug SEGV seen on read of info.id during downsampling```